scipy/sparse/linalg/_expm_multiply.py

```diff
@@ -184,9 +254,14 @@ def _expm_multiply_simple(A, B, t=1.0, balance=False):
         raise ValueError('expected B to be like a matrix or a vector')
     u_d = 2**-53
     tol = u_d
-    mu = _trace(A) / float(n)
+    if traceA is None:  # TODO: we need a good number for m3
+        if is_linear_operator:
+            warn("Trace of LinearOperator not available, it will be estimated."
+                 " Provide `traceA` to ensure performance.")
```

Comment:
Wording is not accurate here and also warnings are missing stacklevel argument.
```suggestion
                 " Provide `traceA` to ensure accuracy and improve performance.",
                 stacklevel=3)
```